scripts/audit_frontend_licenses.py

```diff
@@ -45,6 +45,7 @@
     "CC0-1.0",  # https://creativecommons.org/publicdomain/zero/1.0/
     "CC-BY-3.0",  # https://creativecommons.org/licenses/by/3.0/
     "CC-BY-4.0",  # https://creativecommons.org/licenses/by/4.0/
+    "Python-2.0",  # https://www.python.org/download/releases/2.0/license/
```

Comment:
Added `Python-2.0` to permitted license list per [Snowflake Open Source Policy](https://lift.snowflake.com/$viewer.do?sysparm_stack=no&sysparm_sys_id=b1287f0e1b9c691404ad33fe034bcb84) permissive list.
This is needed as part of this PR for one of `protobufjs-cli`'s transitive dependencies - `argparse`, which is made available under the Python-2.0 license.